### ISSUE TITLE
Client Allocation Request for: YHLL PTY LTD

### ISSUE THREAD

**Issue by YHLLPTY** [filecoin-project/filecoin-plus-client-onboarding#1013]
---

# Client Allocation Questions 

## Core Information
- Name: YHLL PTY LTD
- Website / Social Media: http://ltd.ztlc.org/
- Region: Oceania
- DataCap Requested: 10TiB
- Addresses to be Notarized: f3smuqqn42vw4abb4hm4zunz5wurseqfog5jojsuooxel42cwvlsfo74wruix75ltenhusl3v65enfttg46d3a
- Notary Requested: MegTei

@YHLLPTY Please subscribe to notifications for this Issue to be aware of updates. Notaries may request additional information on the Issue.

**Comment by YHLLPTY** [comment]
@MegTei Thanks.
Australia and Asia.
We know about Filecoin in web3.0 projects, and I saw validation data stores in information. Filecoin is a potential distributed storage network, and Web3.0 is the trend.
YHLL PTY LTD.
Address: Eastlakes, NSW 2018, Australia.
YHLL PTY LTD is an enterprise focusing on the development and application of block technology. The company builds three  major technology scenario business platforms around key technologies such as blockchain, artificial intelligence,  and mobile traffic development,  and jointly explores a new industry ecology. The company is committed to providing artists and creators with a full  range of overseas creation, display,  and service platforms based on blockchain technology for casting NFTs that represent their works. With the help of  blockchain technology,  each digital creation obtained by collectors through YHLL PTY LTD is a unique digital creation signed and issued by the  creator. Even if someone else copies the digital work,  what they get will not be the authentic or original signed version. By connecting artists with art lovers and collectors  through overseas markets and a global network of partners,  artists can better capture the full potential value of digital works.
It is mostly the storage of artists' works. The form of data includes pictures, videos, work manuscripts and records of the creation process.
We have just started to contact. We have a large amount of data. The data volume is expected to be 10 TB per week.
We prefer to permanent storage.
None.
We expect storage providers to provide retrieval services, but retrieval services seem to be in the early stages.
Probably a few times a year.
About five.
No. We didn't start storage provider business.
We agree to provide related information.
